logpyle/__init__.py

```diff
@@ -1309,9 +1309,10 @@ def __call__(self) -> int:
 
 
 class StepToStepDuration(PostLogQuantity):
-    """Records the CPU time between invocations of
-    :meth:`LogManager.tick_before` and
-    :meth:`LogManager.tick_after`.
+    """Records the wall time between the starts of consecutive time steps, i.e.,
+    the wall time between :meth:`LogManager.tick_before` of step x and
+    :meth:`LogManager.tick_before` of step x+1. The value stored is the value for
+    step x+1.
```

Comment:
Out of curiosity, what is the purpose of StepToStepDuration?

Reply:
It records a ground truth: the time between the start of consecutive steps. Ideally, it would match `t_step + t_log`, but if it does not, then it points to an overhead somewhere, or some time that is not being accounted for.